syntax = "proto3";

package driverpacket;

message DriverPacket {
  uint32 sequence_num = 2;
  double rxrate = 3;
  double txrate = 4;
  double txcenterfreq = 5;
  double rxcenterfreq = 6;
  uint32 numberofreceivesamples = 7;
  double seqtime = 8;
  double timetosendsamples = 9;
  bool SOB = 10;
  bool EOB = 11;
  bool align_sequences = 12;
  uint32 buffer_offset = 13;
  uint32 numberoftransmitsamples = 14;
}
